README.md

```diff
@@ -1 +1,23 @@
-# streamlit-whisper
\ No newline at end of file
+# streamlit-whisper
+
+## How to use:
+
+1. Clone repository
```

Comment:
Just to clarify, do we need to install requirements.txt? Also, user need Python installed right?

Reply:
If yes, please add in new section called tools

Reply:
No, all handled by devcontainer